warehouse/templates/manage/settings.html

```diff
@@ -20,6 +20,12 @@
 {% block main %}
   <h2>{% trans %}Project settings{% endtrans %}</h2>
 
+  {% trans project_upload_limit=project.upload_limit %}Project upload limit is: 
+  <a href="https://pypi.org/help/#file-size-limit">{{ project_upload_limit }}{% endtrans %}</a><br><br>
+  
+  {% trans project_total_size_limit=project.total_size_limit %}Project total size limit is: 
+  <a href="https://pypi.org/help/#project-size-limit">{{ project_total_size_limit }}{% endtrans %}</a><br>
+
```

Comment:
The actual values should come from the `project` variable which is available in this template (see how it's used elsewhere in this file)

The help links should be created with `request.help_url` instead.